custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js

```diff
@@ -82,11 +97,11 @@ function MapOrSectorController() {
     };
 }
 
-MapOrSectorController.$inject = [];
+MapOrSectorController.$inject = ['$scope', '$location', 'storageService', 'locationsService'];
```

Comment:
$scope is not used in the code. Please remove it.

Reply:
removed